osm-seed/templates/nominatim-api/nominatim-api-service.yaml

```diff
@@ -30,6 +30,7 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+      targetPort: apache
```

Comment:
@Rub21 is this correct? I'm just not familiar with this, it might be, just double-checking.

Reply:
@batpad , targetPort can be the name or port number that we declare in the statefulset or deployment port section.